### PR TITLE
docs for apache module

### PR DESCRIPTION
**Pull request by td-arash**

### CODE REVIEW

File: agent/deployment/apache/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#this is an optional value which enables verbose logging; the Server's log level should also be set to debug level
+#tdModuleDebugEnabled true 
+
+#this is optional
+#tdModuleServiceContext <service context value provided by tracedata> 
+</IfModule>
+EOL
+```
+
+* Use the `SetInputFilter` and `SetOutputFilter` directives to enable the Tracedata Module for the request and response. Note: We currently only support tracing Request and Request/Response pair only. We do not support tracing just the response by itself. These directives can be used at the Server, VirtualHost, and Directory levels.
```
Comment (marklu2018) [inline agent/deployment/apache/README.md:44]: Where (which config file) to put `SetInputFilter` and `SetOutputFilter`? 

An example or a few example configs would help.

File: agent/deployment/apache/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+SetInputFilter TD_FILTER_REQ
+SetOutputFilter TD_FILTER_RES
+```
+
+* Next we would need to allow the httpd process to make network connection:
+```
+sudo setsebool -P httpd_can_network_connect on
```
Comment (marklu2018) [inline agent/deployment/apache/README.md:53]: `setsebool` is an OS command, not an Apache httpd specific command, right?

File: agent/deployment/apache/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+* After making these configuration changes you need to restart the server.
```
Comment (marklu2018) [inline agent/deployment/apache/README.md:56]: Could you post the command to restart the Apache server? :)

Nginx has one command to reload config changes seamlessly without interrupting existing connections. Is there something similar for Apache?

File: agent/deployment/apache/README.md (hunk 2 above)
Comment (td-arash) [reply agent/deployment/apache/README.md:53]: that is correct.

File: agent/deployment/apache/README.md (hunk 3 above)
Comment (td-arash) [reply agent/deployment/apache/README.md:56]: I use.. sudo systemctl restart httpd
As for hot-restart I am not sure.

File: agent/deployment/apache/README.md (hunk 1 above)
Comment (td-arash) [reply agent/deployment/apache/README.md:44]: Lets talk about this during the call.

**Pull request merged**